### DISCUSSION TITLE
テスト

### DISCUSSION THREAD

**Discussion by ho1125** [URATA-m/-#3] [General]
テスト

**Comment by kanegae-takumi** [comment]
はあっはは

**Comment by URATA-m** [comment]
わーー

**Comment by iwamuro64** [comment]
いぇいいぇい⤴

**Comment by URATA-m** [comment]
お疲れさまです。浦田です。
EclipseとGitHubの連携を行おうと試みたのですが、認証の段階で躓いており先に進めません。
私は用事がありこれ以降は明日の夜まで手を着けられないので、お手すきの方いらっしゃいましたら連携していただけるとありがたいです。
リポジトリのマスターしか連携に必要なコードを入手できないはずなので、個人のリポジトリを新規で作成した上で連携してみてください。
もし連携できた場合は、連携できたリポジトリに招待してください。（全員招待できしだい今使っているこのリポジトリは私の方で削除します。）
力不足でみなさんに迷惑をおかけしてしまい申し訳ありませんが宜しくお願いします。

↓私が連携で参考にしていたサイトのURL添付します。
https://zenn.dev/codek2/articles/0c2d887f98ed01
＊Eclipseで新規作成を行う時は「Javaプロジェクト」ではなく「動的Webプロジェクト」で作成をお願いします。
